src/ni/vsbuild/LabviewBuildVersion.groovy

```diff
@@ -0,0 +1,27 @@
+package ni.vsbuild
+
+class LabviewBuildVersion implements Serializable {
+
+   public static Integer DEFAULT_LABVIEW_BITNESS = 32
```

Comment:
This can be `final`, and probably also an `int`.

Reply:
@buckd this was marked as resolved, but I don't see a change. Am I missing something?

Reply:
@rtzoeller Nope, you didn't miss anything. I ran a replay of a build with that change to make sure it worked, but never actually updated the source. Resolved now.